### DISCUSSION TITLE
POINT_3038

### DISCUSSION THREAD

**Discussion by LissADiaz** [cloudsen12/dataset#215] [Andrea]
20200229T075841_20200229T082630_T35HLE
![image](https://user-images.githubusercontent.com/77083131/147436547-56a72dfb-4459-411a-b619-9c0b025e75a6.png)
![image](https://user-images.githubusercontent.com/77083131/147436600-1b6c2e4a-d2cd-4dcc-88ed-953b3ad3f76b.png)
20190505T075611_20190505T082331_T35HLE
![image](https://user-images.githubusercontent.com/77083131/147436645-aabfc1c9-ba03-4140-b714-b4cdb9556c13.png)
20190321T075619_20190321T081839_T35HLE
![image](https://user-images.githubusercontent.com/77083131/147436769-fca10384-10f3-484d-8556-063d714cc3b1.png)
@andrea29-star

**Comment by andrea29-star** [comment]
este punto no es mio

**Comment by LissADiaz** [comment]
ok,lo corregire
